nexus/catalog/src/lib.rs

```diff
@@ -24,18 +24,24 @@ pub struct Catalog {
 }
 
 async fn run_migrations(client: &mut Client) -> anyhow::Result<()> {
-    let migration_report = embedded::migrations::runner()
+    let migration = embedded::migrations::runner()
         .run_async(client)
         .await
-        .context("Failed to run migrations")?;
-    for migration in migration_report.applied_migrations() {
-        tracing::info!(
-            "Migration Applied -  Name: {}, Version: {}",
-            migration.name(),
-            migration.version()
-        );
+        .context("Failed to run migrations");
+    match migration {
+        Ok(migration_report) => {
+            for migration in migration_report.applied_migrations() {
+                tracing::info!(
+                    "Migration Applied -  Name: {}, Version: {}",
+                    migration.name(),
+                    migration.version()
+                );
+            }
+            Ok(())
+        }
+        Err(err) if err.to_string().contains("duplicate key value violates unique constraint") => Ok(()),
```

Comment:
Problem: this might be too wide in case migration file itself ran into unique constraint violation

Error message I'm looking at:
```
error applying migration V1__initial`, `db error: ERROR: duplicate key value violates unique constraint "refinery_schema_history_pkey"
       DETAIL: Key (version)=(1) already exists.
```
Options:
- more precise string check
- create PR upstream to detect